src/render.jl

```diff
@@ -0,0 +1,19 @@
+"""
+    default_rendermode(::File, actual)
+
+Infer the most appropriate render mode according to type of reference file and `actual`.
+"""
+default_rendermode(::File, ::Any) = BeforeAfterLimited()
```

Comment:
Again, rather than taking the `File` taking the `Format` only seems better

Reply:
done in https://github.com/Evizero/ReferenceTests.jl/pull/37/commits/8bb84a23c3776995332c7cf6091ce1bb1d53b5d9